README.md

```diff
@@ -31,7 +31,7 @@ make test-data
 Run a PostgreSQL instance:
 
 ```
-docker run --rm -it -p 5432:5432 -e POSTGRES_PASSWORD=postgres postgres
+docker run -d --rm --name dashdot-postgres -p 5432:5432 -e POSTGRES_PASSWORD=postgres postgres
```

Comment:
then we miss the console output, please add `docker log dashdot-postgres` as an additional command to show the console outputs :)

Reply:
✅

Reply:
done